doc/sphinx/introduction.rst

```diff
@@ -60,16 +60,18 @@ Nonetheless, the manual has some structure that is explained below.
    of the formalism. Chapter :ref:`themodulesystem` describes the module
    system.
 
--  The second part describes the proof engine. It is divided in six
+-  The second part describes the proof engine. It is divided in seven
```

Comment:
```suggestion
-  The second part describes the proof engine. It is divided into several
```

Better to be nonspecific about the number of chapters--otherwise it may become inaccurate.